protocol/scripts/create_full_node.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+
+if [ $(nproc) -lt 8 ]; then
+  echo "This device has less than 8 cpus, recommended to use a device with at least 8 cpus"
```

Comment:
nit: perform some action after these `echo` - maybe a enter "y" to continue, or something else. otherwise the script runs anyways, will output a bunch of other stuff, and they won't see this message